### PR TITLE
KIALI-2082 Add Policy support to IstioConfig

### PR DESCRIPTION
**Pull request by xeviknal**
** Describe the change **
Users now can manage Policy Istio Objects within IstioConfig section. Users can Update/Delete those policies too.

screen recording 8

** Issue reference **
https://issues.jboss.org/browse/KIALI-2082

** Backwards incompatible? **
Yes.

### CODE REVIEW

**Comment by rhqci** [comment]
Jenkins CI: kiali-core-pr-e2e-test [#420](https://paste.fedoraproject.org/paste/k2qxhopWg-73K5eK7sBEZw)
 :red_circle: PRT Failed, Please Contact QE

**Comment by rhqci** [comment]
Jenkins CI: kiali-core-pr-e2e-test [#421](https://paste.fedoraproject.org/paste/jXHmrFOXCdFfaDrkaYrdnA)
  * :heavy_check_mark: run-kiali-e2e-tests [#[1022]](https://paste.fedoraproject.org/paste/AIoyu5UKJeA3RpiCTomJrQ)

**Comment by rhqci** [comment]
Jenkins CI: kiali-core-pr-e2e-test [#423](https://paste.fedoraproject.org/paste/Hg7RQO1qieKUBCF7a8mTHQ)
  * :heavy_check_mark: run-kiali-e2e-tests [#[1024]](https://paste.fedoraproject.org/paste/ORGzCyIAm3Mr2tJ-zCzDXg)

File: deploy/kubernetes/clusterrole.yaml [hunk 1]
```diff
@@ -74,6 +74,15 @@ rules:
   - list
   - patch
   - watch
+- apiGroups: ["authentication.istio.io"]
```
Comment (lucasponce) [inline deploy/kubernetes/clusterrole.yaml]: cc @jmazzitelli @gbaufake this PR will add new changes in the clusterrole.yaml
I don't remember if we need to perform additional steps in order to propagate this once the PR is merged.

File: deploy/openshift/clusterrole.yaml [hunk 2]
```diff
@@ -74,6 +74,15 @@ rules:
   - list
   - patch
   - watch
+- apiGroups: ["authentication.istio.io"]
```
Comment (lucasponce) [inline deploy/openshift/clusterrole.yaml]: cc @jmazzitelli @gbaufake same here

File: deploy/kubernetes/clusterrole.yaml (hunk 1 above)
Comment (jmazzitelli) [reply deploy/kubernetes/clusterrole.yaml]: We need to create a PR once these changes are in kiali and released as part of v0.14. I created a branch for this - when we are ready for upstream Istio to consume our v0.14 release, I'll create this PR:

https://github.com/istio/istio/compare/master...jmazzitelli:kiali-next-version-changes?expand=1

File: deploy/openshift/clusterrole.yaml (hunk 2 above)
Comment (jmazzitelli) [reply deploy/openshift/clusterrole.yaml]: we need to inform Maistra team about this change. a MAISTRA JIRA would probably be best.

**Pull request merged**